### DISCUSSION TITLE
When you are using it with MMM-Carousel, MMM-CalendarExt2 should be in main page or all pages.

### DISCUSSION THREAD

**Discussion by cactus-george** [eouia/MMM-CalendarExt2#267] [Q&A] [answered]
Hi, as the title suggests, I am trying to use MMM-CalendarExt2 with MMM-Carousel and can't seem to figure out the proper config to allow carousel to rotate between other modules, and one of MMM-CalendarExt2 views.

If it helps, my carousel settings are:

module: "MMM-Carousel",
                        config: {
                                transitionInterval: 10000,
                                mode: "positional",
                                bottom_center: { enabled: true, ignoreModules: [] },
                                bottom_right: { enabled: true, ignoreModules: [] },
                        }
                },

Thanks for your time and assistance.

**Comment by klaernie** [comment]
Did you see this? https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/blob/main/docs/Tip-Sync-with-MMM-Pages.md

**Comment by cactus-george** [comment]
I did not see that.  Is that suggesting CalendarExt2 is not compatible with Carousel, but is with MMM-Pages?

**Comment by klaernie** [comment, marked as answer]
It requires different handling than most modules, since it has its own way. So it isn't compatible with any of the switching modules - but you can control it by sending notifications. And this works with probably all of the switching modules.

**Reply by cactus-george** [reply]
Ok, thanks.  I guess I was confused by the note on the main page about "MMM-CalendarExt2 should be in main page or all pages".
Anyway, thanks again for the prompt reply, assistance, and all your work on this fantastic module.
